packages/core/src/render3/instructions/element.ts

```diff
@@ -190,21 +191,31 @@ export function ɵɵelement(
   return ɵɵelement;
 }
 
-function logUnknownElementError(
-    tView: TView, element: RElement, tNode: TNode, hasDirectives: boolean): void {
-  const schemas = tView.schemas;
-
+/**
+ * Validates that the element is known when running in JIT mode and logs an error if not.
+ *
+ * The element is considered known if either:
+ * - the component was compiled in AOT mode
+ * - the element matches any directive
+ * - the element is allowed by one of the schemas
```

Comment:
I've dropped the AOT point (there is a comment on that above) and added a couple more items.
```suggestion
 * - it's a known HTML element
 * - it's a known custom element 
 * - the element matches any directive
 * - the element is allowed by one of the schemas
```